test/system/grade_books_test.rb

```diff
@@ -72,4 +72,143 @@ class GradeBooksTest < ApplicationSystemTestCase
 
     assert_text "Grade book finalized. Funds have been distributed."
   end
+
+  test "teacher enters grades for multiple students" do
+    visit classroom_grade_book_path(@classroom, @grade_book)
+
+    # Verify students are shown
+    assert_text @student1.username
+    assert_text @student2.username
+
+    # Enter grades for both students
+    within("tbody tr:nth-child(1)") do
```

Comment:
can we avoid using `tr:nth-child(1)`? Having `testid` is better, I think. What do you think?